app/views/govuk_publishing_components/components/_subscription-links.html.erb

```diff
@@ -24,17 +24,19 @@
     >
       <% if sl_helper.email_signup_link.present? %>
         <li class="gem-c-subscription-links__list-item<%= ' gem-c-subscription-links__list-item--small' if local_assigns[:small_form] == true %>" >
+          <svg xmlns="http://www.w3.org/2000/svg" width="21" height="15.75" class="gem-c-subscription-links__icon"><path d="M19.687 0H1.312C.589 0 0 .587 0 1.313v13.124c0 .726.588 1.313 1.313 1.313h18.374c.725 0 1.313-.587 1.313-1.313V1.313C21 .587 20.412 0 19.687 0zm-2.625 2.625L10.5 7.875l-6.563-5.25h13.126zm1.313 10.5H2.625V3.937L10.5 10.5l7.875-6.563v9.188z"/></svg>
```

Comment:
Is it worth adding `role="presentation"` to the SVGs? I know it can be helpful for `img` elements, but I'm unsure about SVG elements.

Reply:
I was intending to put `aria-hidden="true"` to ensure that screen readers ignored them but voiceover on safari ignored them already. I also don't know if `role="presentation"` would be useful or not.

Reply:
I'd also add `focusable="false"` as SVGs are focusable (part of the tab order) by default in Internet Explorer 😕 

You can see how we treat the SVG in the header here:
https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/header/template.njk#L7-L27

Reply:
Bonus points if you put `fill="currentColor"` on the `path` – that way if users override colours in their browser the icons will modify themselves too.

Rather than this:

![Screenshot_2019-07-26 Subscription links component - Component Guide](https://user-images.githubusercontent.com/121939/61938830-e93d4d80-af89-11e9-9035-95842a6565e9.png)

They'll look like this:

![Screenshot_2019-07-26 Subscription links component - Component Guide(1)](https://user-images.githubusercontent.com/121939/61938833-eb071100-af89-11e9-97d2-64d9aeea69ec.png)

Reply:
So many good ideas this morning, thanks guys. I've updated, please take a look.